### PR TITLE
feat(rn): Add Tracing without Performance docs

### PR DESCRIPTION
**Pull request by krystofwoldrich**
- related rn change https://github.com/getsentry/sentry-react-native/pull/3230

## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: src/platform-includes/distributed-tracing/how-to-use/react-native.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+If you're using the current version of our React Native SDK distributed tracing will work out of the box. To get around possible [Browser CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues, define your `tracePropagationTargets`.
```
Comment (AbhiPrasad) [inline src/platform-includes/distributed-tracing/how-to-use/react-native.mdx:1]:
```suggestion
If you're using the current version of our React Native SDK distributed tracing will work out of the box.
```

There's no CORs with React Native right?

**Review by AbhiPrasad — APPROVED** [review]

**Review by shanamatthews — APPROVED** [review]

File: src/platform-includes/distributed-tracing/how-to-use/react-native.mdx (hunk 1 above)
Comment (krystofwoldrich) [reply src/platform-includes/distributed-tracing/how-to-use/react-native.mdx:1]: Yes, that's right. Thanks.

File: src/platform-includes/performance/enable-tracing/react-native.mdx [hunk 2]
```diff
@@ -1,10 +1,8 @@
-<Alert level="Info" title="Prior Versions">
+Tracing is enabled by default in the React Native SDK.
 
-If you use a version of our SDK prior to version `3.0.0`, you will need to include the `ReactNativeTracing` integration to use automatic instrumentation. You do not need to do this in versions `3.0.0` and above.
-
-</Alert>
+<Expandable title="Using older Sentry for React Native? Here is how to enabled Tracing">
 
-<SignInNote />
+If you use a version of our SDK prior to version `3.0.0`, you will need to include the `ReactNativeTracing` integration to use automatic instrumentation. You do not need to do this in versions `3.0.0` and above.
```
Comment (marandaneto) [inline src/platform-includes/performance/enable-tracing/react-native.mdx:5]: We can make `ReactNativeTracing` also a link to the docs that tells you how to set it up

File: src/platform-includes/distributed-tracing/how-to-use/react-native.mdx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracePropagationTargets: [
+    "https://myproject.org",
+    "https://.*.otherservice.org/.*",
+  ],
+});
+```
+
+If you're using version `5.9.x` or below, you'll need to have our <PlatformLink to="/performance/">performance monitoring feature enabled</PlatformLink> in order for distributed tracing to work.
```
Comment (liamjones) [inline src/platform-includes/distributed-tracing/how-to-use/react-native.mdx:13]: Shouldn't this be `5.8.x` or below? The release notes for sentry-react-native 5.9.0 say you don't need perf monitoring

**Comment by liamjones** [comment]
@krystofwoldrich Could I query something extra in relation to this docs update too?

We now have `tracePropagationTargets` at the top-level config of `Sentry.init()` but it looks like it still exists in the options of the `Sentry.ReactNativeTracing` constructor too: https://docs.sentry.io/platforms/react-native/performance/instrumentation/automatic-instrumentation/#tracepropagationtargets

How do the two interact (or not)? E.g. if it's specified at the root can it be skipped from the tracing constructor options and it'll be inherited? Does it need to be set in both places? If specified in both places would the tracing one override the top-level one?

**Pull request merged**